### PR TITLE
Text for the homepage

### PR DESCRIPTION
**Pull request by cryptoguard**
Modifed the text on the homepage, just noticed my repo was late so please verify if nothing broke because of this.

### CODE REVIEW

File: _includes/features.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <section id="features">
-  <h1 class="claim-title">We believe a <span class="highlight">trustless</span>, <span class="highlight">private</span> economy can be built <span class="highlight">without middlemen</span></h1>
+  <h1 class="claim-title"> <span class="highlight">One</span> breakthrough platform, <span class="highlight">endless</span> possibilities</h1>
```
Comment (AllienWorks) [inline _includes/features.html]: I personally prefer the old one, which is more concrete. We should stick to more specific texts instead of these abstract ones IMO.

File: _includes/join-section.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <section class="join-section">
   <img src="/assets/img/favicons/mstile_home.png" class="join-img">
-  <h2 class="title">Join our Riot, Telegram and Discord Channels</h2>
+  <h2 class="title">Join our community and be part of the revolution</h2>
```
Comment (AllienWorks) [inline _includes/join-section.html]: I'd omit the "revolution". Feels like every blockchain project is revolutionary (based on their promo) and it's laughable in majority of cases.. I'd keep our feet on the ground :)

File: _includes/key-points.html [hunk 3]
```diff
@@ -24,10 +24,10 @@ <h2>The swiss-army knife of cryptocurrencies</h2>
     </div>
 
     <div class="medium-6 large-6 columns">
-      <h5 class="lead2">Disruptive E-commerce</h5>
-      <h2>World's most private marketplace</h2>
+      <h5 class="lead2">The Marketplace</h5>
+      <h2>The New Way of Doing eCommerce</h2>
```
Comment (AllienWorks) [inline _includes/key-points.html]: I would highlight the "private" part. "The new way" doesn't necessarity invoke anything better IMO.
Maybe something along the lines of:

> Peer to peer & private approach to e-commerce

?

File: _includes/join-section.html (hunk 2 above)
Comment (cryptoguard) [reply _includes/join-section.html]: Aha yeah, I hesitated about this one and I ultimately agree.

File: _includes/key-points.html (hunk 3 above)
Comment (cryptoguard) [reply _includes/key-points.html]: > Peer to peer & private approach to e-commerce

Sounds awesome for us, but does it really speak to the average user? We should make sure that our message can be something most people can relate to, hence why I'm trying to put more focus on how the marketplace will impact people. Usually, people relate more to saving money than being anonymous, in general. 

Let me think of something else, will post here. 

Thanks for the feedback on these.

Comment (AllienWorks) [reply _includes/key-points.html]: I don't see much problem with calling it what it is (even if technical) in these short titles. I see it as a technical term(s), which are then described "for average users" in the description below. This way it's also a bit educating, if that makes sense.
However if you think of anything better, I'm all for it!

**Comment by cryptoguard** [comment]
@AllienWorks, just updated the PR. Let me know if it's okay and you can see it. Modified the Roadmap.html text.

File: _includes/features.html (hunk 1 above)
Comment (AllienWorks) [reply _includes/features.html]: > We believe a more efficient economy can be built without middlemen

That sound nice :)

File: _includes/join-section.html (hunk 2 above)
Comment (AllienWorks) [reply _includes/join-section.html]: I'd like to keep it short and to the point - maybe remove the channel names, as it's obvious (and listed below in buttons?).. I almost want to have there just:
> Join our community
> Join our passionate community

or
> Join our community of privacy pioneers

File: _layouts/roadmap.html [hunk 4]
```diff
@@ -120,9 +120,9 @@ <h3 class="title">ZMQ for MP Messages</h3>
             <span class="status planned tag">Planned</span>
             <span class="network tag">Testnet/Mainnet</span>
           </div>
-          <h3 class="title">Particl Marketplace Release</h3>
+          <h3 class="title">Mainnet Particl Marketplace Release</h3>
```
Comment (AllienWorks) [inline _layouts/roadmap.html]: Remove the `mainnet` from title - there's already a dedicated "tag" for displaying network (main/test net) above. Let's not mix those.

File: _layouts/roadmap.html [hunk 5]
```diff
@@ -142,9 +142,9 @@ <h3 class="title">Bulletproofs: Short Proofs for Confidential Transactions</h3>
             <span class="status planned tag">Planned</span>
             <span class="network tag">Mainnet</span>
           </div>
-          <h3 class="title">RingCT &ndash; Hardfork</h3>
+          <h3 class="title">Mainnet RingCT Hardfork</h3>
```
Comment (AllienWorks) [inline _layouts/roadmap.html]: Again, remove `mainnet`.

File: _layouts/roadmap.html [hunk 6]
```diff
@@ -29,7 +29,7 @@ <h1 class="title">Particl Project<br />Roadmap</h1>
           </div>
           <h3 class="title">Market Listing Governance</h3>
           <div class="desc">
-            <p>The Particl Marketplace is being built to be self-sustaining, self-governing and completely decentralized. Governance is a huge piece of the puzzle that the team will be focused on developing to protect privacy and protect users.</p>
+            <p>Currently under development and nearing testnet, the Marketplace Governance system will allow users to self-moderate the content listed on Particl Marketplace. This system is entirely controlled by the community and no Particl team member or staff has any authority over the content of the marketplace.</p>
```
Comment (AllienWorks) [inline _layouts/roadmap.html]: Can you please rewrite this, so it's not relative to time? That was probably not clear, so let me explain :)
These roadmap texts should be more descriptions of the items than updates on where we're currently regarding development (that's what the tags above `completed` `in progress` etc. are for). If we'd write it "timing-neutral", we won't have to rewrite it each time some progress is made.. Hope I made myself a bit clearer :)

In short, it means to get rid of the first sentence:
> Currently under development and nearing testnet,

**Comment by AllienWorks** [comment]
@cryptoguard added some more comments & opinions. Let me know what you think.

File: _includes/key-points.html (hunk 3 above)
Comment (AllienWorks) [reply _includes/key-points.html]: If so, please roll back your changes ;)

File: _includes/join-section.html (hunk 2 above)
Comment (cryptoguard) [reply _includes/join-section.html]: > Join our community

Alright then let's keep it like that. I like "of privacy pioneers" as well, but the community is broader than that (ecommerce fans, crypto fans, passive income fans, etc)

**Comment by cryptoguard** [comment]
> @cryptoguard added some more comments & opinions. Let me know what you think.

Somehow never got the notification. Going through this today!

File: _layouts/roadmap.html (hunk 5 above)
Comment (AllienWorks) [reply _layouts/roadmap.html]: Fixed
